packages/shared/src/lib/user.ts

```diff
@@ -43,6 +43,11 @@ export interface UserProfile {
   acceptedMarketing?: boolean;
 }
 
+export type UserShortProfile = Pick<
```

Comment:
We can change this one to `UpvoterProfile` or something that will indicate more explicitly the goal of this type.